doq/_shtab.py

```diff
@@ -0,0 +1,20 @@
+from argparse import Action
+
+FILES = DIRS = {}
+
+
+class PrintCompletionAction(Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        print("Please install shtab firstly!")
```

Comment:
Could you add `noqa: T001 print found` ?
Flake8 was failing.

And prefer quotes are single.